.circleci/config.yml

```diff
@@ -125,6 +128,26 @@ jobs:
           upload_name: Coverage report for iOS functional tests
           xtra_args: -c -v --xc --xp build/reports/iosFunctionalResults.xcresult
   
+  test-ios-integration:
+    macos:
+      xcode: 14.1.0 # Specify the Xcode version to use
+
+    steps:
+      - checkout
+      - when:
+          condition:
+            or:
+              - equal: [ staging, << pipeline.git.branch >> ]
+              - equal: [ dev, << pipeline.git.branch >> ]
```

Comment:
Should we include "main" here as well so the integration tests run when we merge to "main"?